### ISSUE TITLE
How do you use the tuna plugin integration?

### ISSUE THREAD

**Issue by irockdude554** [th-ch/youtube-music#603]
I'm trying to figure out how to use the tuna integration plugin but theres no documentation on how to do so.

**Comment by Araxeus** [comment]
see #372 
(enable webserver option and it should automatically send information to tuna)

**Comment by irockdude554** [comment]
> 

I did that but nothing seems to get sent to my tuna server

**Comment by Araxeus** [comment]
is the server open on port `1608` ?

it should work 😟

**Comment by irockdude554** [comment]
yeah my port is set to 1608 and song source is web browser, but nothing gets sent to tuna.

**Comment by mesmerx** [comment]
again, the tuna happens to not receive POST in some versions, did you try to post do the localhost:1608 to see if this send and receive update?
its not the plugin problem and tuna itself, for arch the only package that works its the bin one

**Comment by irockdude554** [comment]
im on windows, its not tuna as it works with Spotify jsut fine on the default port

**Comment by mesmerx** [comment]
I'll sensd you two requests to you try, the tuna with spotify dont use the server, the server its only for manual requests

**Comment by irockdude554** [comment]
ok

**Comment by mesmerx** [comment]
@irockdude554  sorry the GREAATTT delay man
can you try this request 
```
curl --request POST \
  --url http://localhost:1608/ \
  --header 'Content-Type: application/json' \
  --data '{ "data":{
	"cover_url": "f",
	"title": "ff",
	"artists": [],
	"status": "ff",
	"progress": 0,
	"duration": 0,
	"album_url": "fff",
	"album": "undefined"
}}'
```

if this  next request 

```
curl --request GET \
  --url http://localhost:1608/ \
  --header 'Content-Type: application/json'
```

dont returns 

```
{
	"cover_url": "f",
	"progress": 0,
	"status": "unknown",
	"time_left": 0,
	"title": "ffdsdsd"
}
```

the problem is with the tuna not this plugin :)

**Comment by mesmerx** [comment]
** its not a working payload for the plugin, its just a dummy to know if your server of tuna is working

**Comment by DaNeoPenguin** [comment]
ok problem solved

**Comment by awan-gh** [comment]
When i stop / pause the music, the status of the playback on http://localhost:1608/ just stays on 'playing', even if i close the ymusic it'll still showing the last song title and what not, how can i fix this?

**Comment by mesmerx** [comment]
maybe you can pull a timer to look for the player and see if its in the pause state, the problem its the event for ths

**Comment by JellyBrick** [comment]
> When i stop / pause the music, the status of the playback on [http://localhost:1608](http://localhost:1608/) just stays on 'playing', even if i close the ymusic it'll still showing the last song title and what not, how can i fix this?

Fixed at #1235
